src/modules/settings/components/settings/PreventChatClear.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Panel from 'rsuite/lib/Panel/index.js';
+import Toggle from 'rsuite/lib/Toggle/index.js';
+import {registerComponent, useStorageState} from '../Store.jsx';
+import {CategoryTypes, SettingIds} from '../../../../constants.js';
+import styles from '../../styles/header.module.css'
```

Comment:
Have you run `npm run lint` on this file? looks like there is some formatting issues.

Reply:
the lint script itself is throwing errors for me, but you're right. looked at it again and ran prettier on it's own to make sure it was fixed